cmd/cli/command/deployment/diagnostics.go

```diff
@@ -108,6 +108,110 @@ func JobStateFromString(state string) (diagnosticv1alpha1.JobState, error) {
 	}
 }
 
+type JobSummary struct {
+	Id            string
+	Name          string
+	LastRun       time.Time
+	NextRun       time.Time
+	Errors        string
+	CurrentStatus string
+}
+
+var backgroundTasksGetCommand = cli.Command{
```

Comment:
Rather than a separate command, let's just put this table and JSON output into the `cf deployment diagnostics` command. This means we can tell customers to run a single command, rather than needing to go back and forth and ask them to run multiple different diagnostics commands